coalib/output/ConsoleInteraction.py

```diff
@@ -664,22 +667,22 @@ def choose_action(console_printer, actions, apply_single=False):
                 actions_desc_len = len(actions_desc)
                 if c.isnumeric():
                     for i, action in enumerate(actions, 0):
-                        c = int(c)
-                        if i == c:
+                        if i == int(c):
```

Comment:
The justification for this change was the same as below: I wanted to preserve the choices entered by the user so that invalid choices can be printed with the error message.

Reply:
for preserving `one` you are changing `many` ,if you are concern more related to preserve then kindly store it in different variable and use that ,instead of changing `many`

Reply:
Ahh I see. I'll keep that in mind for the future. Thanks!